platform-emf/src/main/java/com/softicar/platform/emf/form/frame/EmfFormFrameHeader.java

```diff
@@ -14,7 +14,15 @@ public EmfFormFrameHeader() {
 	public EmfFormFrameHeader setCaption(IDisplayString entityTitle, IDisplayString operationTitle) {
 
 		removeChildren();
-		appendText(entityTitle.concat(" - ").concat(operationTitle).toString());
+
+		var captionDiv = appendChild(new DomDiv());
```

Comment:
I think we should have a `private class EmfFormFrameCaption` here - at least that's what we usually do when we have a special CSS class.

Reply:
hm, i looked in `DomPopup` and there is essentially the same thing as this, just a little bit different
![image](https://user-images.githubusercontent.com/90852097/187671680-abb57085-6f72-4d10-bf76-f9c8b3cfede1.png)

Reply:
well, whatever